Students/Cruti Andrei - Marian/Lab04/src/utcn/ro/Order.java

```diff
@@ -0,0 +1,76 @@
+package utcn.ro;
+
+import java.util.Objects;
+
+public class Order {
+    private Integer internalId;
+    private Customer customer;
+    private Product[] products;
+    private Integer[] quantities;
+    private int productIndex = 0;
+
+    public Order(long internalId) {
+        this.products = new Product[6];
+        this.quantities = new Integer[6];
+    }
+
+    public Integer getInternalId() {
+        return internalId;
+    }
+
+    public void setInternalId(Integer internalId) {
+        this.internalId = internalId;
+    }
+
+    public Customer getCustomer() {
+        return customer;
+    }
+
+    public void setCustomer(Customer customer) {
+        this.customer = customer;
+        this.customer.addOrder(this);
+    }
+
+    public void addProduct (Product product, Integer quantity) {
+        for(int i = 0; i < this.productIndex; i++) {
+            Product currentProduct = this.products[i];
+            if(Objects.equals(currentProduct.getInternalId(), product.getInternalId())) {
```

Comment:
spatiu dupa `for, if, while, do, etc.`. Poti folosi auto format din IDE-ul pe care il folosesti. Cauta pe net care e shortcut-ul pentru auto format la cod